frontend/src/js/entity-history/timeline/YearHead.tsx

```diff
@@ -102,20 +124,62 @@ const TimeStratifiedInfos = ({
                   info.columns.findIndex((c) => c.label === l2),
               )
               .map(([label, value]) => {
-                const columnType = getColumnType(info, label);
-                const valueFormatted =
-                  typeof value === "number"
-                    ? Math.round(value)
-                    : value instanceof Array
-                    ? value.join(", ")
-                    : value;
+                const column = info.columns.find((c) => c.label === label);
+
+                if (!column) {
+                  return null;
+                }
+
+                if (isConceptColumn(column)) {
+                  const semantic = column.semantics.find(
+                    (s): s is ColumnDescriptionSemanticConceptColumn =>
+                      s.type === "CONCEPT_COLUMN",
```

Comment:
probably would've wanted to use isConceptColumn?

Reply:
No, here I'm not selecting a column that's a concept column (== that contains at least one semantic of type CONCEPT_COLUMN), but instead I'm selecting the CONCEPT_COLUMN semantic from within the concept column